groups/infrastructure/main.tf

```diff
@@ -0,0 +1,100 @@
+terraform {
+  required_version = ">= 1.3, < 2.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 5.0, < 6.0"
+    }
+    vault = {
+      source  = "hashicorp/vault"
+      version = ">= 4.0, < 5.0"
+    }
+  }
+}
+
+provider "aws" {
+  region  = var.aws_region
+}
+
+terraform {
+  backend "s3" {}
+}
```

Comment:
This `backend` config should be part of the `terraform` block above. This block can then be removed.

Reply:
Done. Pushed an another commit.